Riot/Modules/Room/VoiceMessages/VoiceMessagePlaybackView.swift

```diff
@@ -16,9 +16,13 @@
 
 import Foundation
 import Reusable
+import UIKit
+import MatrixSDK
 
 protocol VoiceMessagePlaybackViewDelegate: AnyObject {
     func voiceMessagePlaybackViewDidRequestPlaybackToggle()
+    func voiceMessagePlaybackViewRequestedFormattedTimestamp(for progress: CGFloat) -> String?
```

Comment:
Let's rename this to didRequest so it's the same as the others.

Reply:
ha yea that was a weird choice on my part